pkg/api/types.go

```diff
@@ -368,6 +368,10 @@ type OpenShiftConfig struct {
 	EnableAADAuthentication bool `json:"enableAADAuthentication,omitempty"`
 
 	ConfigBundles map[string][]byte `json:"configBundles,omitempty"`
+
+	OpenShiftVersion       string
```

Comment:
Thinking out loud: Does a separate `OpenShiftVersion` field need to be added here? Can it use `OrchestratorType` and `OrchestratorVersion` fields? Trying to avoid redundant fields or does `OpenShiftVersion` as a concept doesn't fit with the current semantics of `OrchestratorVersion`?

Reply:
I think this is possible, will remove.